Composer/packages/lib/code-editor/src/LuEditor.tsx

```diff
@@ -16,6 +16,7 @@ import { LUOption } from './utils';
 
 export interface LULSPEditorProps extends BaseEditorProps {
   luOption?: LUOption;
+  helpURL?: string;
```

Comment:
I would recommend create another component `QnAEditor`, instead of modify existed `LuEditor`. Even for now the syntax / parser are same, we may have a customized `QnA LSP Server` or syntax highlighting etc in the future.